samples/log_callback/README.md

```diff
@@ -0,0 +1,114 @@
+# oe_log_message Callback Sample
+
+This sample demonstrates how to customize the log function through the new callback feature of oe_log_message().
+There are several problems with the current logging mechanism that prevent end users from making full use of logging:
```

Comment:
I don't think we need to explain things here, which are already part of the design doc. I'd suggest remove line 4-10, just directly start with Developer Experience that explains how the callback work.

Alternatively, you could specify the version of OE. For example,

This sample demonstrates how to customize the log function through the new callback feature of oe_log_message() introduced in v0.12.

There are several problems with the current logging mechanism (prior to v0.12) that prevent end users from making full use of logging:
...

Reply:
Thanks for the suggestion, yes it is not good to put it in sample. Removed.